turbine/src/cluster_nodes.rs

```diff
@@ -589,10 +644,36 @@ mod tests {
         }
     }
 
```

Comment:
Comment might be useful here clarifying this checks (1) computed retransmit children against expected children and (2) computed parent of each child against expected parent